tw-tasks-core/src/main/java/com/transferwise/tasks/TasksProperties.java

```diff
@@ -352,4 +360,17 @@ public void validate(Object target, Errors errors) {
      */
     private Integer level;
   }
+
+  @Data
+  @Accessors(chain = true)
+  public static class Environment {
+
+    /**
+     * Version deployed (e.g. to production).
```

Comment:
I see some risk for confusion here too. Is there an expectation that service owners will continually keep updating not only the tw-tasks dependency version, but the previous version? I believe not, but that means this information will go stale soon.

Reply:
Yep, it is expected to go stale. And then with some new version, the service does not start anymore and they can then change it.